src/components/cards/next-event-card/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Flex, Text } from "@radix-ui/themes";
+import Image from "next/image";
+import Link from "next/link";
+
+import { INextEvent, padToStartNumber } from "@/src/constants/next-events-mock";
+
+export const NextEventCard: ({
+  type,
+  title,
+  link,
+  date,
+  img,
+  time,
+}: INextEvent) => JSX.Element = ({
+  type,
+  title,
+  link,
+  date,
+  img,
+  time,
+}: INextEvent) => {
+  function formatDateWithDayName(date: Date): string {
+    const daysOfWeek: string[] = [
+      "Sunday",
```

Comment:
Why the first day of the week is Sunday?

Reply:
done